harper-core/src/linting/initialisms.rs

```diff
@@ -118,4 +118,13 @@ mod tests {
             "Really excited for this.",
         );
     }
+
+    #[test]
+    fn issue_2181() {
+        assert_suggestion_result(
+            "AFAIK, we don't currently have an issue for it.",
+            lint_group(),
+            "As far as i know, we don't currently have an issue for it.",
```

Comment:
Ah I see what you mean. That's a bit unfortunate. Maybe we can have a function that converts a run of text to "normal" case, similar to how we have `title_case.rs`
A beginner version could do nothing other than capitalize every instance of the word "i".
A more advanced version could do a lookup on every word and look at the orth flags to try to work out of a word has a single canonical lettercase spelling.